chart/compass/values.yaml

```diff
@@ -16,7 +16,7 @@ global:
       version: "PR-251"
     healthchecker:
       dir: pr/
-      version: "PR-99"
+      version: "PR-287"
```

Comment:
Please reset change for healthchecker image. Unfortunately we cannot have PR-XXX images here.